ppa/pages/templates/pages/content_page.html

```diff
@@ -26,6 +27,8 @@
 <article class="ui text container">
     <h1 class="ui header">
         {% firstof page.title page_title object.title %}
+        {% block attribution %}{% endblock %}
```

Comment:
This seems a little weird to me, but I don't feel strongly about it. Could make more sense to wrap the header contents in a `header` block that can be extend with `super`.

Reply:
This was actually so that the attribution line could fit in the `h1` header as as to be above the horizontal rule bar. Wrapping it would make inserting there hard, unless I'm missing something (totally likely)?

Reply:
Oh, I didn't catch that. The attribution shouldn't be inside the `<h1>`, should it? Just the title? Maybe we need a `<header>` or something. But I don't feel that strongly about it; if the markup is provisional, we can leave for @thatbudakguy to sort out when he finishes the styles.

Reply:
@bwhicks is exactly right; it's weird to put the attribution inside the `<h1>` but I do it so that I can extend the border below it to the correct length. this might all change because I didn't notice that the border lines are supposed to run past the margin on the right side, and I also didn't do them in a faded color because borders can't have opacity, so I might have to rework how they're rendered entirely.